src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
```

Comment:
Consider making the 'random' variable final, since it is not being modified after initialization. This helps ensure the variable remains constant throughout the class lifecycle 🎯